CMakeLists.txt

```diff
@@ -369,18 +369,8 @@ endif()
 # Reset the required flags
 set(CMAKE_REQUIRED_FLAGS ${OLD_CMAKE_REQUIRED_FLAGS})
 
-CHECK_CXX_SOURCE_COMPILES("
-#if defined(_MSC_VER) && !defined(__thread)
-#define __thread __declspec(thread)
-#endif
-int main() {
-  static __thread int tls;
-  (void)tls;
-}
-" HAVE_THREAD_LOCAL)
-if(HAVE_THREAD_LOCAL)
-  add_definitions(-DROCKSDB_SUPPORT_THREAD_LOCAL)
-endif()
+# Thread-local is part of C++11 (TODO: clean up this flag)
```

Comment:
C++17 or part of the C++ standard.  (It is a bit confusing to say C++11 when we say we support 17 only.

Reply:
C++11 is factually accurate. I'm pointing out that the check has been effectively obsolete for a long time.